### PR TITLE
Adding contact form

### PR DESCRIPTION
**Pull request by edgardsoza**
1. Contact form was added. 
2. Code was cleaned by my learning partner. 
3. Form is active and sending request messages to my mail. 
4. Used developer tools in order to make sure the page does not have scroll effects.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -164,10 +164,14 @@ <h2 class="titleekills">Skills</h2>
...
-            <input class="email-address" type="email" placeholder="Email address"></br>
-            <input class="enter-text" type="text" placeholder="Enter text here"></br>
-            <button enable="enable" class="get-in-button" type="action">Get in touch</button>
+            <form action="https://formspree.io/f/xeqwoznl" method="post">
+                <ul class="contact-form">
+                    <li><input type="text" id="name" name="full_name" placeholder="   Full Name" maxlength="30" required/></li>
+                    <li><input type="email" id="email" name="contact_email" placeholder="   Email address" required/></li>
+                    <li><textarea id="message" name="contact_message" placeholder="   Enter text here" maxlength="500" required></textarea>
+                    <li class="submit_button"><button type="submit">Get in touch</button></li>
+                </ul>
```
Comment (huseyinbiyik) [inline index.html]: Good job with the validations! But let's fix the small design issues, please add padding to left side of input areas like on the template below.
Screenshot 2023-01-24 at 11 17 14 PM
Screenshot 2023-01-24 at 11 17 59 PM

File: index.html [hunk 2]
```diff
@@ -164,10 +164,14 @@ <h2 class="titleekills">Skills</h2>
...
-            <input class="full-name" type="text" placeholder="Full Name"></br>
-            <input class="email-address" type="email" placeholder="Email address"></br>
-            <input class="enter-text" type="text" placeholder="Enter text here"></br>
-            <button enable="enable" class="get-in-button" type="action">Get in touch</button>
+            <form action="https://formspree.io/f/xeqwoznl" method="post">
+                <ul class="contact-form">
+                    <li><input type="text" id="name" name="full_name" placeholder="   Full Name" maxlength="30" required/></li>
+                    <li><input type="email" id="email" name="contact_email" placeholder="   Email address" required/></li>
+                    <li><textarea id="message" name="contact_message" placeholder="   Enter text here" maxlength="500" required></textarea>
+                    <li class="submit_button"><button type="submit">Get in touch</button></li>
```
Comment (huseyinbiyik) [inline index.html]: We have enabled/default state for our submit button. Let's add states: pressed and hover like on the Figma template.
Screenshot 2023-01-24 at 11 22 46 PM

File: index.html [hunk 3]
```diff
@@ -180,4 +184,4 @@ <h4 class="my-email">jose.sozasobalvarro@gmail.com</h4>
         </div>
```
Comment (huseyinbiyik) [inline index.html:184]: Please follow the Figma template for styling the mail address. Font size and family don't match. Kindly check the differences below and fix it. 👨‍🎨
Screenshot 2023-01-24 at 11 22 03 PM

Screenshot 2023-01-24 at 11 21 17 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @edgardsoza and @everinurmind,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Form submission works fine 👌
- Linter set up well and checks pass 🟢
- Github flow followed 👌

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @edgardsoza  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [ ] Implemented all the changes requested by the previous reviewer :white_check_mark: 
- [ ] Descriptive pull request title and summary :white_check_mark: 
- [ ] Professional readme file :white_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**